### ISSUE TITLE
Incorrectly eaten value (tables)

### ISSUE THREAD

**Issue by brettz9** [wooorm/mdast#97]
I'm getting the error, "Incorrectly eaten value" with the following tabular code:

```
# Rules

| ColA | ColB | ColC | ColD | ColE | ColF | ColG |
| - | - | - | - | - | - | - |
| a | b | c | d | | f  | g  |
```

It even has problems if I fix it to have the three hyphens needed per Github (but not my Atom README previewer):

```
# Rules

| ColA | ColB | ColC | ColD | ColE | ColF | ColG |
| --- |  --- |  --- |  --- |  --- |  --- |  --- |
| a | b | c | d | | f  | g  |
```

Here it is in preview form:

# Rules

| ColA | ColB | ColC | ColD | ColE | ColF | ColG |
| --- |  --- |  --- |  --- |  --- |  --- |  --- |
| a | b | c | d | | f  | g  |

The culprit seems to be column E.

**Comment by wooorm** [comment]
Hey @brettz9, thanks for the detailed report!

Could you confirm whether you’re on the latest version of mdast? [I thought I fixes this](https://github.com/wooorm/mdast/commit/baad5362bb774cba5d56452f2736b6fab657179b).

**Comment by brettz9** [comment]
Oh, apologies--I am getting it through AtomLinter/linter-markdown, so I guess I should just report a request to update there... Thanks!

**Comment by wooorm** [comment]
Yes, definitely!  Thanks again for taking the time :+1: Also: [this is the spot that should be changed in linter-markdown](https://github.com/AtomLinter/linter-markdown/blob/a14bdcb9129cd41c5d97cd6c6cc3abc9561800d8/package.json#L14-L15).

**Comment by brettz9** [comment]
Yes, thank you, I've made a PR: https://github.com/brettz9/linter-markdown/pull/1

**Comment by Arcanemagus** [comment]
@brettz9 looks like you submitted the PR against your own fork instead of the main one so nobody saw it, it's already fixed on the primary one and released as v1.2.2 though.

**Comment by brettz9** [comment]
Yeah, sorry. :)
